### PR TITLE
Fix some styles in Product Finder

### PR DESCRIPTION
**Pull request by jose-correia**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/product-finder
- After: https://styles-product-finder--moleculardevices--hlxsites.hlx.page/product-finder

Screenshot 2023-07-25 at 15 50 46

Screenshot 2023-07-25 at 15 50 55

Screenshot 2023-07-25 at 15 50 59

Screenshot 2023-07-25 at 15 51 05

### CODE REVIEW

**Comment by alexcarol** [comment]
Related (though not necessarily for this PR): the link in "read more" redirects to www.moleculardevices.com, rather than the franklin site.

**Comment by alexcarol** [comment]
I still see the "compare button" with one product:
image

**Comment by alexcarol** [comment]
In the live site the arrow at the end of the product title is also a link to the product.

**Review by mhaack — APPROVED** [review]

File: blocks/product-finder/product-finder.js [hunk 1]
```diff
@@ -216,12 +216,12 @@ async function stepThree(e) {
     });
     const cardRenderer = await createCard({
       c2aLinkStyle: true,
-      defaultButtonText: 'Request Quote',
+      defaultButtonText: 'Read More',
     });
     products.forEach((product) => {
       product.c2aLinkConfig = {
         href: `https://www.moleculardevices.com/quote-request?pid=${product.familyID}`,
```
Comment (mhaack) [inline blocks/product-finder/product-finder.js:223]: This should be relativ URL only `/quote-request?pid=${product.familyID}` should be fine

**Review by lydiapuric — APPROVED** [review]
+1

**Pull request merged**